crystal-map-connector/src/main/kotlin/ch/ergon/dope/extension/type/arithmetic/NumberInfix.kt

```diff
@@ -31,6 +35,9 @@ fun Number.sub(number: CMField<out Number>) = toDopeType().sub(number.toDopeType
 
 fun CMField<out Number>.sub(number: Number) = toDopeType().sub(number.toDopeType())
 
+fun <KotlinType : Any, MapType : Number> KotlinType.sub(other: CMConverterField<KotlinType, MapType>) =
```

Comment:
What about if _other_ has type _CMField_?

Reply:
Good point. I completely forgot about this case

Reply:
Thats not possible, because we wouldn't have the typeconverter in the CMField